packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -436,11 +484,22 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
     return new ExpressionPriceFeed(priceFeedMap, expressionConfig.expression, expressionConfig.priceFeedDecimals);
   }
 
-  // Returns a MedianizerPriceFeed
   async function _createMedianizerPriceFeed(medianizerConfig) {
-    const priceFeedsToMedianize = [];
-    for (const _priceFeedConfig of medianizerConfig.medianizedFeeds) {
-      // The medianized feeds should inherit config options from the parent config if it doesn't define those values
+    const priceFeedsToMedianize = await _createConstituentPriceFeed(medianizerConfig.medianizedFeeds);
+    if (!priceFeedsToMedianize) return null;
+    return new MedianizerPriceFeed(priceFeedsToMedianize, medianizerConfig.computeMean);
+  }
+
+  async function _createFallBackPriceFeed(fallbackConfig) {
+    const orderedPriceFeeds = await _createConstituentPriceFeed(fallbackConfig.orderedFeeds);
+    if (!orderedPriceFeeds) return null;
+    return new FallBackPriceFeed(orderedPriceFeeds);
+  }
+
+  async function _createConstituentPriceFeed(priceFeedConfigs) {
```

Comment:
Would plural make sense considering it takes in and returns an array?

```suggestion
  async function _createConstituentPriceFeeds(priceFeedConfigs) {
```